websites/docs.temporal.io/src/theme/EditThisPage/index.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+
+export default function EditThisPageWrapper() {
+  const feedbackStyle = {
+    marginTop: "20px",
+    padding: "10px 15px",
+
+    borderRadius: "8px",
+    borderLeft: "4px solid #007bff", // A blue accent to match the button
+    display: "flex",
+    flexDirection: "column",
+    alignItems: "flex-start",
+  };
+
+  return (
+    <>
+      <div style={feedbackStyle}>
+        <p style={{ margin: 0, fontWeight: "500" }}>Have feedback on our documentation?</p>
+        <a href="https://temporalio.slack.com/ssb/redirect" target="_blank" rel="noopener noreferrer">
+          Join our community Slack channel
+        </a>
```

Comment:
```suggestion
        Post in the #documentation channel in our
        <a href="http://t.mp/slack" target="_blank" rel="noopener noreferrer">
          community Slack
        </a>
        or read the 
        <a href="https://github.com/temporalio/documentation/blob/main/README.md#how-to-make-changes-to-this-repository" target="_blank" rel="noopener noreferrer">
          contributing guide.
        </a>
```

> https://temporalio.slack.com/ssb/redirect

link doesn't work unless already a member of our Slack. t.mp/slack is an invite link so non-members can join

> We can provide a pinned walkthrough in Slack on how to contribute.

Not everyone pays attention to pinned posts. Wdyt about linking to a walkthrough in a markdown file on github as well?